nautobot/extras/plugins/views.py

```diff
@@ -29,6 +29,17 @@ def load_marketplace_data():
     return marketplace_data
 
 
+def get_app_headline(package_name, app_configs, marketplace_data):
+    """If the given package_name is in the marketplace_data, use its headline there; else read from app_configs."""
+    for marketplace_app in marketplace_data["apps"]:
+        if marketplace_app["package_name"] == package_name and marketplace_app["headline"]:
+            return marketplace_app["headline"]
+    for app_config in app_configs:
+        if app_config.name == package_name:
+            return app_config.description
```

Comment:
I reviewed all of the function calls and it seems like you can just pass `app_config.name, app_config.description, marketplace_data` and then remove this for loop.